src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,22 @@
+package lotto.model;
+
+import java.util.List;
+import lotto.policy.LottoPolicy;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        new LottoPolicy().checkLottoPolicy(numbers);
```

Comment:
```suggestion
        LottoPolicy.checkLottoPolicy(numbers);
```
checkLottoPolicy()를 static으로 선언하고 위에 같이 써도 좋을 것 같아요! 아님 따로 저렇게 작성하신 이유가 있을까요?